utils/token_cache.py

```
@@ -0,0 +1,334 @@
+            common_texts: Dictionary of {description: text} for warming
+            model_names: List of models to warm cache for (default: ["default"])
+        """
+        if model_names is None:
+            model_names = ["default"]
+
+        from utils.token_utils import estimate_tokens  # Import here to avoid circular import
+
+        warmed_count = 0
+
+        with self._lock:
+            for desc, text in common_texts.items():
```

Ruff (0.11.9)

Loop control variable `desc` not used within loop body

Rename unused `desc` to `_desc`

(B007)